alacritty_terminal/src/term/mod.rs

```diff
@@ -501,6 +501,57 @@ impl<T> Term<T> {
         }
     }
 
+    /// Go to next prompt farthest down in history.
+    pub fn goto_next_prompt(&mut self)
+    where
+        T: EventListener,
+    {
+        let point = if self.mode.contains(TermMode::VI) {
+            self.vi_mode_cursor.point
+        } else {
+            let display_offset = self.grid.display_offset() as i32;
+            // If we're outside of Vi mode use bottom most visible line.
```

Comment:
```suggestion
            // If we're outside of Vi mode, use bottommost visible line.
```

Topmost/bottommost are one word, as weird as it looks.